src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Formatting/RazorFormattingService.cs

```diff
@@ -240,4 +241,20 @@ private static void UnwrapCSharpSnippets(TextEdit[] snippetEdits)
             snippetEdit.NewText = unwrappedText;
         }
     }
+
+    // This method counts the occurrences of CRLF and LF line endings in the original text. If LF line endings are more
+    // prevalent, it removes any CR characters from the text edits to ensure consistency with the LF style.
+    // This can be removed once we figure out how to generate the formatted CSharp file with the correct line endings.
```

Comment:
If it's still relevant after other changes you're planning to make, consider converting this comment to an XML doc comment.